Sources/AppsFlyerConstants.swift

```diff
@@ -130,4 +298,75 @@ public enum AppsFlyerConstants {
         static let error = "appsflyer_error"
     }
     
+    public enum MediationNetwork: String, CaseIterable {
+        case ironsource = "ironsource"
+        case applovinmax = "applovinmax"
+        case googleadmob = "googleadmob"
+        case fyber = "fyber"
+        case appodeal = "appodeal"
+        case admost = "admost"
+        case topon = "topon"
+        case tradplus = "tradplus"
+        case yandex = "yandex"
+        case chartboost = "chartboost"
+        case unity = "unity"
+        case toponpte = "toponpte"
+        case custommediation = "custommediation"
+        case directmonetizationnetwork = "directmonetizationnetwork"
+
+        public static func fromString(_ value: String) -> MediationNetwork? {
+            return MediationNetwork(rawValue: value.lowercased())
+        }
+        
+        // Convert to AppsFlyerLib's MediationNetworkType
+        public func toAppsFlyerType() -> MediationNetworkType {
+            switch self {
+            case .ironsource:
+                return MediationNetworkType.ironSource
+            case .applovinmax:
+                return MediationNetworkType.applovinMax
+            case .googleadmob:
+                return MediationNetworkType.googleAdMob
+            case .fyber:
+                return MediationNetworkType.fyber
+            case .appodeal:
+                return MediationNetworkType.appodeal
+            case .admost:
+                return MediationNetworkType.admost
+            case .topon:
+                return MediationNetworkType.topon
+            case .tradplus:
+                return MediationNetworkType.tradplus
+            case .yandex:
+                return MediationNetworkType.yandex
+            case .chartboost:
+                return MediationNetworkType.chartBoost
+            case .unity:
+                return MediationNetworkType.unity
+            case .toponpte:
+                return MediationNetworkType.toponPte
+            case .custommediation:
+                return MediationNetworkType.custom
+            case .directmonetizationnetwork:
+                return MediationNetworkType.directMonetization
+            }
+        }
+        
+        // Convenience method to convert string directly to AppsFlyerLib type
+        public static func appsFlyerTypeFromString(_ value: String) -> MediationNetworkType {
+            return fromString(value)?.toAppsFlyerType() ?? MediationNetworkType.googleAdMob
```

Comment:
```suggestion
            return MediationNetwork(rawValue: value.lowercased())?.toAppsFlyerType() ?? MediationNetworkType.googleAdMob
```
Btw, perhaps you could use the same mapping approach as you did with `standardEventMapping` function.

Reply:
I like the idea of a dictionary mapping instead of this triple conversion here. 
Arguably you could also avoid specifying an enum for our values: Just go with `[String: MediationNetworkType]` and simplify a lot. 
What we did [here](https://github.com/Tealium/tealium-ios-firebase-remote-command/blob/main/Sources/FirebaseRemoteCommand.swift#L303) is Firebase, for example, works perfectly and it's very clear.

Reply:
Now we are giving customer posibility to user MediationNetwork.ironsource in apps, instead of creating their own class using ours 'strings', but the question is, should we care?

Reply:
@sebastian-krajna I don't know if the comment was directed to me, but I don't understand what you mean.